packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -75,6 +93,14 @@ export default {
 		this.destroyPopper()
 	},
 	methods: {
+		clickHandler(option) {
+			if (
+				!option.disabled &&
```

Comment:
Should be `.isDisabled` if this is a new API

Reply:
ok